deel-2-flexbox-layout/opdracht-1/styles/styles.css

```diff
@@ -1 +1,48 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+body {
+    height: 966px;
```

Comment:
min-height: 100vh; means that the minimum height of an element should be 100% of the viewport height